src/renderer/components/ft-create-playlist-prompt/ft-create-playlist-prompt.js

```diff
@@ -33,26 +33,45 @@ export default defineComponent({
     newPlaylistVideoObject: function () {
       return this.$store.getters.getNewPlaylistVideoObject
     },
+
+    playlistNameEmpty() {
```

Comment:
**note:** once the Vue 3 migration is complete, this will be a great place for composable functions.